jenkins/stretchtests.jpl

```diff
@@ -0,0 +1,12 @@
+@Library('kernelci') _
+import org.kernelci.debian.rootfs
+
+def k = new rootfs()
+
+def config = ['name':"stretchtests",
+              'arch_list':["armhf", "armel", "arm64", "i386", "amd64", "mips", "mipsel", "mips64el"],
+              'debian_suite':"stretch",
+              'extra_packages':"libpciaccess0 libkmod2 libprocps6 libcairo2 libssl1.1 libunwind8 libudev1 libglib2.0-0 libdw1 liblzma5",
+              'script':"scripts/stretchtests.sh"]
+
+k.buildImage (config)
```

Comment:
Ditto about `k`.  Also I think the usual coding style for Groovy is like Java, with no space between a function/method name and the brackets.

Reply:
:+1: